c2corg_api/models/cache_version.py

```diff
@@ -295,3 +299,19 @@ def update_cache_version_associations(
             text('SELECT guidebook.update_cache_version_of_routes(:route_ids)'),  # noqa: E501
             {'route_ids': list(routes_to_update)}
         )
+
+
+def get_cache_key(document_id, lang):
+    version = DBSession.query(CacheVersion.version). \
+        filter(CacheVersion.document_id == document_id). \
+        first()
+
+    if not version:
+        # no version for this document id, the document should not exist
+        log.debug('no version for document id {0}'.format(document_id))
+        return
+
+    if not lang:
```

Comment:
Getting a document with `/{type}/{id}` is a valid request for the API though I think it is not used in the UI.